python-json-rbac/config.py

```
@@ -2,16 +2,107 @@
 # --- Secrets / Keys ----------------------------------------------------------
 JWT_SECRET: str = os.getenv("JWT_SECRET", "")
+JWT_SECRET_PREVIOUS: str = os.getenv("JWT_SECRET_PREVIOUS", "")  # For key rotation
+
+# Validate primary secret
 if not JWT_SECRET:
     raise RuntimeError("JWT_SECRET environment variable is required")
 
+try:
+    _validate_secret_strength(JWT_SECRET)
+except ValueError as e:
+    raise RuntimeError(f"JWT_SECRET validation failed: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)